modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -161,12 +161,6 @@ public void onWrite(Iterator<CommandClosure<WriteCommand>> iterator) {
             try {
                 processWriteCommand(clo);
             } catch (Throwable t) {
-                LOG.error(
```

Comment:
If `clo.result(t)` throws an exception itself, is it guaranteed that it will log `t` itself? It seems more reliable to still have this `LOG.error()` in the beginning: if we duplicate logging, it creates a little noise in the log, but if we miss it completely, we'll not know about the exception at all

Reply:
Yes, it's now logger on upper level in the DelegatingStateMachine